examples/boolqa/README.md

```diff
@@ -0,0 +1,209 @@
+# Training a full TyDiQA model
+
+Here we describe how to train the TyDiQA model `PrimeQA-Reader-with-Boolean` which was submitted to the TyDiQA leaderboard on 11/1/2022.
+This model has full support for boolean questions.  For simplified inference-only TyDiQA with suppport for boolean questions using earlier models,
+please see [here](../../primeqa/boolqa/README.md).
+Training this model is a multistage process:
+
+- first we customize a general-purpose MRC model for TyDiQA
+
+- then we train a question type classifier that determines if questions are short answer or yes/no.
+
+- then we train an evidence classifier that decides how the yes/no quesitons should be answered
+
+- then we train a score normalizer so that the answerable/non-answerable distinctions are reflected in the confidence score,
+independent of question type.
+
+- Finally we integrate these models into the configuration file so that the `do_boolean` option in `run_mrc.py` can use them.
+
+### Adapting the MRC component
+We use as a starting point an MRC model trained on NQ, TyDiQA, and SQuAD.  Support for training this model is coming soon.
```

Comment:
We use as a starting point an XLM Roberta Large MRC model trained on NQ, TyDiQA, and SQuAD.  Support for training this model is coming soon.